flag_engine/features/schemas.py

```diff
@@ -14,7 +14,7 @@
 class FeatureSchema(Schema):
     id = fields.Int()
     name = fields.Str()
-    type = fields.Str()
+    type = fields.Str(allow_none=True)
```

Comment:
Not sure why this is needed or why it's crept in to a PR related to traits?

Reply:
>Not sure why this is needed

because it's not a  required in django feature model.

>why it's crept in to a PR related to traits?

That's a good question